### ISSUE TITLE
ImportError: cannot import name 'WEIGHTS_NAME' from 'pytorch_pretrained_bert.file_utils'

### ISSUE THREAD

**Issue by lcswillems** [huggingface/pytorch-pretrained-BERT#523]
I just tried to run `run_squad.py` example and I got this error:

```
Traceback (most recent call last):
  File "run_squad.py", line 37, in <module>
    from pytorch_pretrained_bert.file_utils import PYTORCH_PRETRAINED_BERT_CACHE, WEIGHTS_NAME, CONFIG_NAME
ImportError: cannot import name 'WEIGHTS_NAME' from 'pytorch_pretrained_bert.file_utils' (/mnt/Data/miniconda3/lib/python3.7/site-packages/pytorch_pretrained_bert/file_utils.py)
```

**Comment by ahmetgunduz** [comment]
Same is happening for `run_classifier.py`

**Comment by thomwolf** [comment]
Yes the examples currently requires to install from source (see the section in the readme).
I'll release a new version tomorrow so the pip release will be in sync with `master` examples again.

**Comment by lcswillems** [comment]
Okay, thank you :)

**Comment by bikashg** [comment]
Waiting for this; installing from source gives the error : `ImportError: cannot import name 'warmup_linear'`

**Comment by ahmetgunduz** [comment]
> Waiting for this; installing from source gives the error : `ImportError: cannot import name 'warmup_linear'`

It is not actually using warm_up so you can safely remove that from the file

**Comment by thomwolf** [comment]
Ok, I've just published and uploaded the new v0.6.2 release on pip which should fix this (among other things). Release notes are [here](https://github.com/huggingface/pytorch-pretrained-BERT/releases/tag/v0.6.2).
